### ISSUE TITLE
Bonfire: Search and Replace code test calling replace instead of myReplace

### ISSUE THREAD

**Issue by platypusrex** [FreeCodeCamp/FreeCodeCamp#3614]
http://www.freecodecamp.com/challenges/bonfire-search-and-replace

Looks like the code test for this activity is calling replace instead of myReplace. This through me off for a bit as I was getting the expected output from my code. 

![screen shot 2015-10-06 at 11 03 33 pm](https://cloud.githubusercontent.com/assets/7143612/10328149/b2c37d9e-6c7e-11e5-8656-10b7dbb95620.png)

Just in case the screen shot isn't enough...here's my code.

~~~
function myReplace(str, before, after) {  
  return str.replace(before, function(){
   if(before[0] === before[0].toUpperCase()){
     return after.charAt(0).toUpperCase() + after.slice(1);
   }else {
     return after;
   }
 });
}
~~~

**Comment by jlslew** [comment]
There is a similar issue; #3271 and a fix has been made; #3326.

**Comment by bugron** [comment]
@platypusrex thanks for posting this issue!
Actually this is fixed in `staging` branch. Hopefully, the changes will be pushed to production soon. Closing this issue.

>***
>Please review the [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), thank you!.
>***
